<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RemoveDialog</class>
 <widget class="QDialog" name="RemoveDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>329</width>
    <height>173</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Remove Stuff</string>
  </property>
  <widget class="QPushButton" name="Finish_pushButton">
   <property name="geometry">
    <rect>
     <x>41</x>
     <y>89</y>
     <width>85</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Finish</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Cancel_pushButton">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>90</y>
     <width>85</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>41</x>
     <y>51</y>
     <width>236</width>
     <height>34</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string> Remove ID:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="ID_lineEdit"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
